aspnetcore/fundamentals/websockets/samples/2.x/WebSocketsSample/EchoApp.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" />
```

Comment:
Use the `App` package ... `Microsoft.AspNetCore.App`

Reply:
Oop,  I forgot this one